### ISSUE TITLE
BUG: create service: parse route options: parse DNS server[dns-local]: unknown DNS server format: underlying://0.0.0.0

### ISSUE THREAD

**Issue by saremeskandary** [MatsuriDayo/nekoray#1449]
## Describe the problem

Expected behavior:
ssh routing should work as like as previous releases.

Actual behavior:
it shows error `create service: parse route options: parse DNS server[dns-local]: unknown DNS server format: underlying://0.0.0.0`

## How to reproduce
set custom outbound
```
{
    "client_version": "SSH-2.0-OpenSSH_7.4p1",
    "password": "",
    "server": "",
    "server_port": 22,
    "type": "ssh",
    "user": "root"
}

**Comment by 17kato** [comment]
how fix this? 
create service: parse route options: parse DNS server[dns-local]: unknown DNS server format: underlying://0.0.0.0

**Comment by Geil** [comment]
Same on 4.0-beta4.
Will revert to the release version.

**Comment by Arctomachine** [comment]
I get this error after updating to stable v4.
Using vless by url string from 3xui

**Comment by lexx63** [comment]
Same after updating to stable v4.

Fixed by manually downloading latest release and setting everything from scratch.

**Comment by ilyxa769** [comment]
Most likely you have an old client running after the update. Try to go to the root folder and launch the new client, if you launched it earlier via shortcut

**Comment by Arctomachine** [comment]
Is it nekoray.exe file? It gives this error.

**Comment by olegchir** [comment]
Just run nekobox.exe instead of nekoray.exe

And stop all the running clients

**Comment by Arctomachine** [comment]
It works. But why did it change required file?

**Comment by NikunoTaro** [comment]
> Most likely you have an old client running after the update. Try to go to the root folder and launch the new client, if you launched it earlier via shortcut很可能你还在运行旧客户端。尝试进入根文件夹并启动新客户端，如果你之前通过快捷方式启动过它。


Yes, that's the reason. The correct solution is to open the root directory of NekoBox, then delete the old one and start it with the new exe.

**Comment by persamplex** [comment]
For Android devices just **clear cache and data**
